src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -10800,7 +10849,7 @@ private static IEnumerable<object> NewMethod(ref IEnumerable<object> enumerable)
             // allowMovingDeclaration: false is default behavior on VS. 
```

Comment:
Should the comment here also be removed?

Reply:
Yup!  It def should be. :)